src/main/java/com/google/firebase/remoteconfig/RemoteConfigParameter.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.remoteconfig;
+
+import com.google.firebase.remoteconfig.internal.TemplateResponse.ParameterResponse;
+import com.google.firebase.remoteconfig.internal.TemplateResponse.ParameterValueResponse;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class RemoteConfigParameter {
+
+  private RemoteConfigParameterValue defaultValue;
+  private String description;
+  private Map<String, RemoteConfigParameterValue> conditionalValues;
+
+  public RemoteConfigParameter() {
+    conditionalValues = new HashMap<>();
+    defaultValue = ExplicitParameterValue.of("");
```

Comment:
Why this default value?

Reply:
If no default value is set (if `defaultValue` is `null`), the REST API sets the default value to an empty string. We can allow `null` here and add a `null` check in the conversion or just set it to an empty value in the `ctor`.